osu.Framework.Android/AndroidGameView.cs

```
@@ -242,6 +242,21 @@ public override IInputConnection OnCreateInputConnection(EditorInfo outAttrs)
             return new AndroidInputConnection(this, true);
         }
 
+        public override void SwapBuffers()
+        {
+            try
+            {
+                base.SwapBuffers();
+            }
+            catch
+            {
+                // sometimes buffers will spontaneously fail to swap just before the activity is suspended to background
+                // or just after it has been resumed, but will continue operating correctly after that transitionary period.
+                // despite some testing it is unclear which view callback can be used to tell whether it is safe to swap buffers,
+                // so for now just catch and suppress errors.
```

While suppressing, I think it may still be better logging the thrown exception in case we come across another android-related issue where a failing `SwapBuffers` call is relevant (especially if it isn't a `BAD_SURFACE` error)

---

Can probably type this to `GraphicsContextException` and check `message` to only handle the specific ones we care about.

Not sure about logging, depends how many times it fires. If it was logged it shouldn't be `Logger.Error`, but a message mentioning that x exception was suppressed.

---

Yeah `Logger.Log` at verbose level is what I had in mind, for the main purpose of showing on GitHub/sentry reports without irritating the user.

---

I've considered scoping the suppression more but decided to start simple and see where it goes at review time. I'll make the suggested changes.

---

I've made the suggested changes. Please see if they match your expectations.